words.txt

```diff
@@ -0,0 +1,192 @@
+addi
+Basefee
+basefee
+bijective
+Bitfield
+bitfield
+bitlist
+bitlists
+BLOBBASEFEE
+blobbasefee
+blobdata
+Blockhash
+blockhash
+blockheaders
+blockquotes
+Blockspace
+blockspace
+blocksv
+blocktime
+Bootnode
+bottlenecked
+Brotli
+bytestring
+CALLVALUE
+cellborder
+CHAINID
+chainid
+checkpointed
+clabby
+codehash
+Coef
+configurables
+counterfactually
+cwia
+datas
+Decentralizable
+delegatees
+delegators
+Dencun
+Depositer
+devnet
+dialable
+Discv
+discv
+disincentivized
+DOCTOC
+doctoc
+EBADF
+Ectone
+Edgington
+elif
+ENDIANNESS
+errno
+fanout
+Fastlz
+fastlz
+fcntl
+Filecoin
+FINDNODES
+flippin
+Forkchoice
+forkchoice
+FPAC
+FPVM
+froms
+frontran
+gaspayingtoken
+gaspayingtokenname
+gaspayingtokensymbol
+GETFL
+gindex
+griefed
+Hardfork
+hardfork
+hardforks
+hardfork’s
+idxs
+IERC
+Impls
```

Comment:
Looks like this is being pulled from [a code block](https://github.com/ethereum-optimism/specs/blob/992df6bf62f4ab1f6181bd07cae2b55e3ba0b577/specs/fault-proof/stage-one/dispute-game-interface.md?plain=1#L166)?  Does the `ignoreRegExpList` regex need to be updated?